source/extensions/common/dynamic_forward_proxy/dns_cache_impl.h

```diff
@@ -14,9 +14,29 @@ namespace Extensions {
 namespace Common {
 namespace DynamicForwardProxy {
 
+/**
+ * All DNS cache stats. @see stats_macros.h
+ */
+#define ALL_DNS_CACHE_STATS(COUNTER, GAUGE)                                                        \
+  COUNTER(dns_query_attempt)                                                                       \
+  COUNTER(dns_query_success)                                                                       \
+  COUNTER(dns_query_failure)                                                                       \
```

Comment:
abc order? All the ones I messed with in a big PR a few weeks ago I made abc-order.